ratelimit_test.go

```diff
@@ -125,6 +126,32 @@ var takeTests = []struct {
 	}},
 }}
 
+var availTests = []struct {
+	capacity     int64
+	fillInterval time.Duration
+	take         int64
+	sleep        time.Duration
+
+	expectCountAfterTake  int64
+	expectCountAfterSleep int64
+}{
```

Comment:
}{{

to save an indent level.

Also, please move the tests to just before TestAvailable.